cli/src/main.rs

```diff
@@ -238,7 +235,7 @@ pub fn main() {
         unflagged_lwd_log.push_str(lwd_logs.to_str().expect("error making lwd_datadir"));
         unflagged_lwd_log.push_str("lwd.log");
 
-        let lwd_command = Command::new(lwd_bin)
+        let _lwd_command = Command::new(lwd_bin)
```

Comment:
This variable should be removed, it's being bound but the invocation is for the side-effect.

Reply:
Wasn't sure, so I erred on the side of not changing the semantics. Will do